cylc/flow/scheduler.py

```diff
@@ -583,13 +594,25 @@ def load_tasks_for_run(self):
             if self.config.start_point is None:
                 # No start cycle point at which to load cycling tasks.
                 continue
-            try:
-                self.pool.add_to_runahead_pool(TaskProxy(
-                    self.config.get_taskdef(name), self.config.start_point,
-                    is_startup=True))
-            except TaskProxySequenceBoundsError as exc:
-                LOG.debug(str(exc))
+            tdef = self.config.get_taskdef(name)
+            spawn = []
+            for seq in tdef.sequences:
+                point = seq.get_first_point(self.config.start_point)
+                if point is None:
+                    continue
+                spawn.append([tdef, self.config.start_point, point])
```

Comment:
`tdef` and `self.config.start_point` don't change within this loop.

```python

start_point = self.config.start_point
spawn = list(sorted((
    point
    for seq in tdefs.sequences
    for point in [seq.get_first_point(start_point)]
    if point
)))
```

Reply:
Good spotting, but I don't much like the `for point in [scalar]` hack, and I find it easier to get my head around this variant (which I'm pretty sure gives the same result):
```python
spawn = sorted([
    point
    for point in (seq.get_first_point(self.config.start_point)
                      for seq in tdef.sequences) if point
    ])
```
Trying this...